package.json

```diff
@@ -46,6 +46,7 @@
     "grunt-dojo2": "beta1",
     "grunt-tslint": "^5.0.1",
     "intern": "^3.4.1",
+    "intersection-observer": "^0.2.1",
```

Comment:
This is only a dev dependency?  It feels like we are using a polyfill here.  This feels like we are introducing something fundamental as a dependency, like pointer events.  We should have a more formal discussion about introducing something like this.  While the [platform status seems](https://www.chromestatus.com/feature/5695342691483648) like it is headed the right direction, we still need to proactively make a decision.